python/flink_agents/api/event.py

```diff
@@ -19,30 +19,19 @@
 from typing import Any
 from uuid import UUID, uuid4
 
-from pydantic import BaseModel, Field, model_validator
+from pydantic import BaseModel, Field
 
 
 class Event(BaseModel, ABC, extra="allow"):
-    """Base class for all event types in the system. Event allow extra properties, but
-    these properties are required isinstance of BaseModel, or json serializable.
+    """Base class for all event types in the system.
 
     Attributes:
     ----------
     id : UUID
         Unique identifier for the event, automatically generated using uuid4.
     """
-    id: UUID = Field(default_factory=uuid4)
-
-    @model_validator(mode='after')
-    def validate_extra(self) -> 'Event':
-        """Ensure init fields is serializable."""
-        self.model_dump_json()
-        return self
 
-    def __setattr__(self, name: str, value: Any) -> None:
-        super().__setattr__(name, value)
-        # Ensure added property can be serialized.
-        self.model_dump_json()
```

Comment:
Not sure about removing these. If this is because some types currently won't pass the validation, we can bypass those types and leave a todo item.